python/ray/services.py

```diff
@@ -303,6 +304,7 @@ def start_redis(node_ip_address="127.0.0.1", port=None, num_retries=20,
   redis_client = redis.StrictRedis(host="127.0.0.1", port=port)
   # Wait for the Redis server to start.
   wait_for_redis_to_start("127.0.0.1", port)
+  redis_client.config_set("maxclients", "65536")
```

Comment:
Maybe we should do this in a separate PR so we can get it merged right away.

Also, this kind of thing may throw an exception, so it should be inside of a `try` `except` block. Also, maybe we should try something like

```
try:
  for val in range(12, 17):
    redis_client.config_set("maxclients", str(2 ** val))
except:
  ...
```

so that we get the maximum number of clients that we can get.